AppKit/CPEvent.j

```diff
@@ -202,6 +226,28 @@ var _CPEventPeriodicEventPeriod         = 0,
 }
 
 /* @ignore */
+- (id)_initEnterExitEventWithType:(CPEventType)anEventType location:(CGPoint)aPoint modifierFlags:(unsigned)modifierFlags
```

Comment:
Since the only method this is called from is `enterExitEventWithType`, there is no point in having a separate method.